src/frontend/apps/desk/src/features/header/Header.tsx

```diff
@@ -55,9 +56,9 @@ export const Header = () => {
               alt={t('Freedom Equality Fraternity Logo')}
             />
             <Box $align="center" $gap="1rem" $direction="row">
-              <Image priority src={IconDesk} alt={t('Desk Logo')} />
+              <Image priority src={IconApplication} alt={t('Equipes Logo')} />
               <Text className="m-0" as="h2" $theme="primary">
-                {t('Desk')}
+                {t('Equipes')}
               </Text>
```

Comment:
Is it intended to not write _Équipes_ ? :nerd_face:

Reply:
I followed Johann's design, it feels weird to have an **é** in English?